.evergreen/config.yml

```diff
@@ -530,12 +530,10 @@ functions:
           export GOFLAGS=-mod=vendor
           AUTH="auth" \
           SSL="ssl" \
-          MONGODB_URI="${SINGLE_ATLASPROXY_SERVERLESS_URI}" \
+          MONGODB_URI="${SERVERLESS_URI}" \
           SERVERLESS="serverless" \
           SERVERLESS_ATLAS_USER="${SERVERLESS_ATLAS_USER}" \
           SERVERLESS_ATLAS_PASSWORD="${SERVERLESS_ATLAS_PASSWORD}" \
-          SINGLE_MONGOS_LB_URI="${SINGLE_ATLASPROXY_SERVERLESS_URI}" \
```

Comment:
I created a serverless instance to test the code in this PR and ran the following test:

```
go test -run TestUnifiedSpec/change-streams/unified/change-streams.json/Test_that_comment_is_not_set_on_getMore_-_pre_4.4 -v
```

and am getting this error:
```
SINGLE_MONGOS_LB_URI must be set when running against load balanced clusters
```
This was the URI I exported:
```
export MONGODB_URI=mongodb+srv://28987-drivertest.3x79d.mongodb-dev.net
```